SODA.tex

```diff
@@ -654,6 +654,47 @@ \subsubsection{POL}
 The UCD  describing the POL parameter is
 \ucd{meta.code;phys.polarization}.
 
+\subsubsection{SPATRES}
```

Comment:
These are currently under the filtering parameters subsection (3.3) Should they instead be under a new subsection such as transformation parameters ?